src/index.html

```diff
@@ -20,8 +23,396 @@
       href="styles/index.scss"
     />
   </head>
+  <body class="page__body">
+    <header class="header">
+      <a
+        class="header__logo-link"
+        href="index.html"
+      >
+        <img
+          class="header__logo"
+          src="./images/logo.png"
+          alt="MOYO logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="nav__item">
```